<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenster</class>
 <widget class="QWidget" name="fenster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Spieler1PaareAnzeige">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>textl</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler2PaareAnzeige">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>820</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler3PaareAnzeige">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>90</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler4PaareAnzeige">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>810</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler1NameAnzeige">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Text Semibold'; font-size:28pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#ff00ff;&quot;&gt;Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler2NameAnzeige">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>760</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler3NameAnzeige">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Spieler4NameAnzeige">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>750</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic2_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>1201</width>
     <height>591</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="GewinnerButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>881</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>72</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ReifenLabel1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ReifenLabel1_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>740</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ReifenLabel1_3">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>740</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ReifenLabel1_4">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>10</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LorBeerBlattLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LorBeerBlattLabel_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>290</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SpielerAmZugAnzeige">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Text Semibold'; font-size:28pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Spieler am Zug:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="SpielerAmZugAnzeige_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>281</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI Variable Text Semibold'; font-size:28pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>pic2_label</zorder>
  <zorder>GewinnerButton</zorder>
  <zorder>ReifenLabel1</zorder>
  <zorder>Spieler1NameAnzeige</zorder>
  <zorder>Spieler1PaareAnzeige</zorder>
  <zorder>ReifenLabel1_2</zorder>
  <zorder>ReifenLabel1_3</zorder>
  <zorder>ReifenLabel1_4</zorder>
  <zorder>Spieler4NameAnzeige</zorder>
  <zorder>Spieler4PaareAnzeige</zorder>
  <zorder>Spieler3NameAnzeige</zorder>
  <zorder>Spieler3PaareAnzeige</zorder>
  <zorder>Spieler2NameAnzeige</zorder>
  <zorder>Spieler2PaareAnzeige</zorder>
  <zorder>LorBeerBlattLabel</zorder>
  <zorder>LorBeerBlattLabel_3</zorder>
  <zorder>SpielerAmZugAnzeige</zorder>
  <zorder>SpielerAmZugAnzeige_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
